lib/stat_tracker.rb

```diff
@@ -36,6 +37,23 @@ def sum_game_goals
     game_goals_hash
   end
 
+  def wd_total_goals(filtered_games = @games)
+    filtered_games.reduce(0) do |sum, game|
+      sum += (game.home_goals + game.away_goals)
+    end
+  end
```

Comment:
To make this more descriptive, change from wd_total_goals to total_goals_as_integer

Reply:
Hi Jake, changes made.  Though I just called it total_goals.  Is that cool?